manifest.toml

```
@@ -0,0 +1,69 @@
+packaging_format = 2
+
+id = "vaultwarden"
+name = "Vaultwarden"
+description.en = "Manage passwords and other sensitive informations"
+description.fr = "Gérez les mots de passe et autres informations sensibles"
+
+version = "1.29.2~ynh1"
```

ah my bad, main brain ships `.1`, not `.2`, hmm ?

```suggestion
version = "1.29.1~ynh1"
```